tests/iam_policies/user-role.cfn.yaml

```diff
@@ -823,10 +823,12 @@ Resources:
               - dynamodb:BatchWriteItem
             Resource: !Sub arn:${AWS::Partition}:dynamodb:${AWS::Region}:${AWS::AccountId}:table/parallelcluster-*
             Effect: Allow
-          - Action: ec2:DescribeInstances
```

Comment:
Why do we need to convert all these to lists?  Does this affect current tests?

Reply:
No this doesnt affect the test. I just converted it to a list so that when I am trying to parse this through a template generator and make my changes, I can use a generic functionality rather than creating another function to handle it as all the other policy statements have been a list so far.

Reply:
I hav already tested the changes using jenkins